### DISCUSSION TITLE
DRM Tools haven't worked for me for a few months now

### DISCUSSION THREAD

**Discussion by bethgaut** [apprenticeharper/DeDRM_tools#1606] [Q&A]
I had the latest Calibre software and tools but can no longer decrypt my Kindle books, so I changed back to Calibre 4.23have reverted back  Calibre 4.23 and DeDRM 6.8.1 and still cannot decrypt.

This is my Log:
calibre Debug log
calibre 4.23  embedded-python: True is64bit: False
Windows-10-10.0.19041-SP0 Windows ('32bit', 'WindowsPE')
32bit process running on 64bit windows
('Windows', '10', '10.0.19041')
Python 2.7.16+
Windows: ('10', '10.0.19041', 'SP0', u'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (6, 8, 0) && Reading List (1, 7, 2) && Obok DeDRM (6, 7, 0)
calibre 4.23  embedded-python: True is64bit: False
Windows-10-10.0.19041-SP0 Windows ('32bit', 'WindowsPE')
32bit process running on 64bit windows
('Windows', '10', '10.0.19041')
Python 2.7.16+
Windows: ('10', '10.0.19041', 'SP0', u'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (6, 8, 0) && Reading List (1, 7, 2) && Obok DeDRM (6, 7, 0)
Turning on automatic hidpi scaling
devicePixelRatio: 1.0
logicalDpi: 120.0 x 120.0
physicalDpi: 141.76744186 x 142.134715026
Using calibre Qt style: True
[0.00] Starting up...
[0.00] Showing splash screen...
[0.55] splash screen shown
[0.55] Initializing db...
[1.73] db initialized
[1.73] Constructing main UI...
DEBUG:    0.0 obok::utilities.py - loading translations
DEBUG:    0.0 obok::dialogs.py - loading translations
DEBUG:    0.0 obok::config.py - loading translations
DEBUG:    0.0 obok::action_err.py - loading translations
[4.83] main UI initialized...
[4.83] Hiding splash screen
[19.88] splash screen hidden
[19.88] Started up in 19.88 seconds with 18770 books
stdout+stderr from file dialog helper: ['', '']
piped data from file dialog helper: ['\xcb\x1c\x10$B\xc3\x9a\xb4\xbf\xae5\x0fQ?\x17\xd48J\x955J\xe2\x9cl\xb3W94\xb6\xc9\xf7\x86', 'C:\\Users\\Beth\\Documents\\My Kindle Content\\B01BVU88IS_EBOK\\B01BVU88IS_EBOK.azw']
DeDRM v6.8.0: Trying to decrypt B01BVU88IS_EBOK.azw
Using Library AlfCrypto DLL/DYLIB/SO
DeDRM v6.8.0: Failed to decrypt with error: The .kfx DRMION file cannot be decrypted by itself. A .kfx-zip archive containing a DRM voucher is required.
DeDRM v6.8.0: Looking for new default Kindle Key after 0.1 seconds
searching for kinfoFiles in C:\Users\Beth\AppData\Local
Found K4PC 1.9+ kinf2011 file: C:\Users\Beth\AppData\Local\Amazon\Kindle\storage\.kinf2011
Decrypted key file using IDString '3833084591' and UserName '42657468'
DeDRM v6.8.0: Found 1 new key
DeDRM v6.8.0: Ultimately failed to decrypt after 0.2 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "site-packages\calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 626, in run
  File "calibre_plugins.dedrm.__init__", line 576, in KindleMobiDecrypt
DeDRMError: DeDRM v6.8.0: Ultimately failed to decrypt after 0.2 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Traceback (most recent call last):
  File "site-packages\calibre\customize\ui.py", line 428, in get_file_type_metadata
  File "site-packages\calibre\customize\builtins.py", line 285, in get_metadata
  File "site-packages\calibre\ebooks\metadata\mobi.py", line 498, in get_metadata
  File "site-packages\calibre\ebooks\mobi\reader\headers.py", line 295, in __init__
  File "site-packages\calibre\ebooks\mobi\reader\headers.py", line 323, in identity
UnicodeDecodeError: 'utf8' codec can't decode byte 0x81 in position 0: invalid start byte
Added B01BVU88IS EBOK to db in: 1.8
Added 1 books in 2.6 seconds

**Comment by ElleKayEm** [comment]
This is a KFX file.  You need the KFX Input plugin.  Or disable KFX downloads.  See this post for details:  https://www.mobileread.com/forums/showthread.php?t=283371

**Reply by bethgaut** [reply]
Thank you! I'm back in business!
